### ISSUE TITLE
Sharing a training?

### ISSUE THREAD

**Issue by niekjuuhh** [iperov/DeepFaceLab#746]
My question is can you share a training to use on a other pc. Besides that is it possible to let 2 PC's run 1 video?

**Comment by The-Digital-Void** [comment]
Yes, if you copy the workspace you can train on two PCs. You can train one video on 2 PCs as well, but not the same frames.

If you have two faces you can train one on each PC.

If you have one face you can train front-facing frames on one, while training more extreme angles on the other. If you do this you'll want to combine the merged frames when you convert to the result video.

**Comment by niekjuuhh** [comment]
Okay and is it possible to use a already trained face? For example i have my src trained on my dst. Can i put my trained src on a new dst?

**Comment by The-Digital-Void** [comment]
You can do this in a way, but not completely. If you take the model file "decoder_dst.npy" you can rename it to "decoder_src.npy" and add it into the new model file. This will give you a head-start on the new training but you won't start from where you left off when using it as a DST. Back up your model and experiment to see how it affects your times, but that would probably be your best bet and you'll still need to train.

**Comment by ghost** [comment]
This is not an issue (or related to deepfacelab), please close it.
